build/Targets/Imports.targets

```diff
@@ -128,6 +126,17 @@
       />
   </Target> 
 
+  <Target Name="DisableAnalyzers"
+          BeforeTargets="CoreCompile"
+          Condition="'$(UseRoslynAnalyzers)' == 'false'">
```

Comment:
The analyzers are enabled by default, correct? If so, how about negating the condition and calling the property `SuppressRoslynAnalyzers`?

Reply:
💭 It seems unnecessary. The property has always had this meaning prior to this pull request and I just kept it.

Reply:
Agreed with Sam, I'd prefer to avoid seemingly unnecessary changes like this.

Reply:
ok

Reply:
Especially in this case this is a property passed by a bunch of other CI scripts and such: if we want to invert the meaning we should do that explicitly and not when we're trying to unbreak the world.